ring-core/src/ring/middleware/not_modified.clj

```diff
@@ -26,15 +26,26 @@
 (defn- ok-response? [response]
   (= (:status response) 200))
 
+(defn- validators-match?
+  "Checks if all provided cache validators match against the current
```

Comment:
Can you remove the docstring? It's a private function so it's unnecessary.

Reply:
I added the docstring to make it clearer what the intention of the code is. Do you want it as a comment, or not at all?

Reply:
Not at all, I think. Docstrings on private functions are (more or less) the same as comments. If you're reading them, there's a good chance you have the source code open.

Comments should, in my view, explain *why* rather than *what*. If it's not obvious *why* we've written some code, it deserves a comment to explain. On the other hand, *what* a function does should be obvious from the code; if it's not obvious, then we need to write clearer code!